tests/lib/rules/jest/no-vague-titles.js

```diff
@@ -593,11 +630,15 @@ ruleTester.run('no-tests-contain-correct with ignore=describe', rule, {
       code: "describe.only('appropriate')",
       options: [{ignore: ['describe']}],
     },
+    {
+      code: `describe.each([['production'], ['staging']])('Includes things all for %s clients')`,
```

Comment:
Doesn’t include any of the blacklisted words?

Reply:
Oh, I guess all is restricted, seems weird. In any case, the valid and invalid examples are much too subtle imo.